src/main/java/use_cases/add_edit_category_use_case/CategoryUCI.java

```diff
@@ -0,0 +1,146 @@
+package use_cases.add_edit_category_use_case;
+
+import entities.Category;
+import entities.EntityException;
+import entities.MonthlyStorage;
+
+import java.util.ArrayList;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
+/**
+ * The CategoryUCI adds/creates new category, edits an existing category, and updates MonthlyStorage.categoryData.
+ * Contains a helper method isValidDouble() and findCategory().
```

Comment:
isValidDouble changed names and needs updating in the JavaDoc

Reply:
Or actually, I don't know if you even need to write this here? They should probably be private methods and not be mentioned here